rotkehlchen/api/rest.py

```diff
@@ -4597,7 +4606,7 @@ def add_addressbook_entries(
     def update_addressbook_entries(
             self,
             book_type: AddressbookType,
-            entries: list[AddressbookEntry],
+            entries: list[tuple[ChecksumEvmAddress, str, Optional[SupportedBlockchain]]],
```

Comment:
Since you are repeating this type in several places maybe you can declare this as a type and reuse it